loki_operator.md

```diff
@@ -130,5 +130,58 @@ Check [ZeroSSL.com CA with acme.sh](./hack_dex.md#zerosslcom-ca-with-acmesh)
 
 ```bash
 oc exec -it netobserv-plugin-d894b4544-97tq2 -- curl --cert /var/loki-status-certs-user/tls.crt  --key /var/loki-status-certs-user/tls.key  --cacert /var/loki-status-certs-ca/service-ca.crt -k -H "X-Scope-OrgID: network"  https://loki-query-frontend-http:3100/loki/api/v1/label/DstK8S_Namespace/values
+```
+
+### Loki "input size too long" error
+
+This error is generally seen when non cluster-admin users have access to many namespaces. Since the operator gateway will inject namespaces in queries for
+multi-tenancy, this may result in queries being too long, reaching 5120 characters (which sounds actually like a very reasonable limit for non-cluster-admin users), and thus being rejected by Loki.
+
+#### For cluster admins
+
+Users identified as cluster admins should not see this error because there is no namespace-based restriction for them. The Loki operator performs this identification by **checking if users belong to one of the defined cluster-admin groups**.
+
+Note that having the cluster-admin **role** does not imply belonging to the cluster-admin **group**. So you should double-check groups.
+
+With OpenShift, to create a cluster-admin group, run:
+
+```bash
+oc adm groups new cluster-admin
+```
+
+To add a user to cluster-admin group, run:
+
+```bash
+oc adm policy add-cluster-role-to-user cluster-admin <username>
+```
+
+You may also want to give cluster-admin role for all group members rather than one by one:
+
+```bash
+oc adm policy add-cluster-role-to-group cluster-admin cluster-admin
+```
+
+Running these commands should solve the problem. It might be necessary to wait a couple of minutes for changes to be effective.
```

Comment:
Is there a way to see when these changes are reloaded ? 
I wonder if restarting the gateway pods doesn't make it instant :thinking:

Reply:
Yes killing the gateway would work .. but that also means potentially lost flows from FLP, I think it's ok to just say to wait a bit? :-)
That's because the gateways are caching the permission checks, I don't remember exactly how long it's kept